components/atoms/Avatar/avatar.tsx

```diff
@@ -20,19 +20,26 @@ const avatarLoader = () => {
 };
 
 const Avatar = (props: AvatarProps): JSX.Element => {
-  const imageSource =
+  if(typeof props.avatarURL === "string") new URL(props.avatarURL);
```

Comment:
IMHO this is not doing anything?

Reply:
`new URL(...)` will throw an error if the string inside isn't a proper url. 
![image](https://user-images.githubusercontent.com/52013393/225723969-36ea8f63-75b4-4bfb-a671-d8290f3d6945.png)

Reply:
I don't think we should thrown runtime errors that aren't caught and handled

Reply:
I agree with ted on this, the check is not leveraged anywhere in the component

```
try {
    // Checks if the avatarURL is a proper URL. If not, it will throw an error.
    if (typeof imageSource === "string") new URL(imageSource);
  } catch (error) {
    console.error(error);
    imageSource = undefined;
  }
```
The above line does the work IMHO

Reply:
Oh, I understand now, yeah that's definitely a mistake